src/css/neo4j-docs.css

```diff
@@ -13,13 +13,35 @@ section.deprecated .title::after {
   margin-left: 10px;
 }
 
-body.alpha article h1::after,
-body.beta article h1::after,
+div.page-roles,
+div.page-roles > span.page-role {
+  display: inline-flex;
+  margin: 0 0 1.5rem;
+  font-size: calc(36 / var(--rem-base) * 1rem);
```

Comment:
Just curious: why the 36 here? Magic number? Or does `rem-base` always divide 36?

Reply:
It's here because I copied an existing css rule to make sure the font size was right for these labels :) I think the magic is in the `--rem-base var` - if it is altered, elements that calc based on it will be auto-scaled, including heights and widths of things like toc and navigation.